src/cli/cli_network_data.cpp

```diff
@@ -354,6 +354,29 @@ template <> otError NetworkData::Process<Cmd("publish")>(Arg aArgs[])
         error = otNetDataPublishExternalRoute(GetInstancePtr(), &config);
         ExitNow();
     }
+
+    /**
+     * @cli netdata publish replace
+     * @code
+     * netdata publish replace ::/0 fd00:1234:5678::/64 s high
+     * Done
+     * @endcode
+     * @cparam publish replace #ca{oldprefix} @ca{prefix} [@ca{sn}] [@ca{high}|@ca{med}|@ca{low}]
```

Comment:
Should start with `netdata publish replace`

And first parameter should be `@ca` not `#ca`

Reply:
Thanks